src/transformers/models/sam/modeling_sam.py

```diff
@@ -1270,6 +1270,34 @@ def forward(
         return_dict=None,
         **kwargs,
     ) -> List[Dict[str, torch.Tensor]]:
+        r"""
+        Returns:
+
+        Example:
+
+        ```python
+        >>> from PIL import Image
+        >>> import requests
+        >>> from transformers import AutoModel, AutoProcessor
+
+        >>> model = AutoModel.from_pretrained("facebook/sam-vit-base")
+        >>> processor = AutoProcessor.from_pretrained("facebook/sam-vit-base")
+
+        >>> img_url = "https://huggingface.co/ybelkada/segment-anything/resolve/main/assets/car.png"
+        >>> raw_image = Image.open(requests.get(img_url, stream=True).raw).convert("RGB")
+        >>> input_points = [[[400, 650]]]  # 2D location of a window on the car
+        >>> inputs = processor(images=raw_image, input_points=input_points, return_tensors="pt")
+
+        >>> # Get segmentation mask
+        >>> outputs = model(**inputs)
+        >>> iou_scores = outputs["iou_scores"]
```

Comment:
they are not used and you don't expect them so can be removed

Reply:
Makes sense!